Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clean.py

```diff
@@ -114,3 +115,51 @@ def main(cls, args, repository, **kwargs):
         for argument in args.arguments:
             result += cls.cleanup(repository, argument, remote_target=args.remote)
         return result
+
+
+class CleanPRBranches(Command):
+    name = 'clean-pr-branches'
```

Comment:
probably `delete-pr-branches` would be more clear to the users than `clean-pr-branches`?